modules/ossm-federation-create-export.adoc

```diff
@@ -11,6 +11,11 @@ You create an `ExportedServiceSet` resource to explicitly declare the services t
 
 Services are exported as `<export-name>.<export-namespace>.svc.<ServiceMeshPeer.name>-exports.local` and will automatically route to the target service.  This is the name by which the exported service is known in the exporting mesh. When the ingress gateway receives a request destined for this name, it will be routed to the actual service being exported. For example, if a service named `ratings.red-mesh-bookinfo` is exported to `green-mesh` as `ratings.bookinfo`, the service will be exported under the name `ratings.bookinfo.svc.green-mesh-exports.local`, and traffic received by the ingress gateway for that hostname will be routed to the `ratings.red-mesh-bookinfo` service.
 
+[NOTE]
+====
+When you set the `importAsLocal` parameter to `true` to aggregate the remote endpoint with local services you must use an alias for the service. When you set the parameter `false` no alias is required.
```

Comment:
Up to you but I think a comma here would be a helpful pause for the reader:
```suggestion
When you set the `importAsLocal` parameter to `true` to aggregate the remote endpoint with local services, you must use an alias for the service. When you set the parameter `false` no alias is required.
```

Reply:
Made this edit.